docs/articles/retrieval_augmented_generation_eval_qdrant_arize.md

```diff
@@ -0,0 +1,526 @@
+
+This Article will take you through :
+
+- Learning about key concepts and metrics in Phoenix
+- Evaluating a Naive RAG using Phoenix
+- Key observations
+- Building a [Hybrid RAG]([https://superlinked.com/vectorhub/articles/optimizing-rag-with-hybrid-search-reranking](https://superlinked.com/vectorhub/articles/optimizing-rag-with-hybrid-search-reranking))
+- Evaluating a Hybrid RAG using Phoenix
+- Concluding thoughts
+
+Lets begin!
+
+**Learning about key concepts and metrics in Phoenix**
+
+Phoenix is an open-source observability library designed for experimentation, evaluation, and troubleshooting. It allows AI Engineers to quickly visualize their data, evaluate performance, track down issues and optimize.
+
+![../assets/use_cases/retrieval_augmented_generation_eval_qdrant_arize/arize_products.png](../assets/use_cases/retrieval_augmented_generation_eval_qdrant_arize/arize_products.png)
+
+It provides a platform to enable evaluation and observability across different layers and stages of an LLM-based system.
+
+Arize offers a comprehensive suite of solutions to meet various needs, from open-source to enterprise-grade offerings. This [LLM tracing](https://docs.arize.com/phoenix/tracing/llm-traces-1) application provides visibility into the topology of your application whether you're running it locally in Jupyter notebook or a Docker container, or pushing your application into staging and production, the solutions provide unparalleled observability into every aspect of your LLM system.
+
+This includes tracing, prompt iteration, search and retrieval, [evaluating specific tasks](https://arize.com/blog-course/llm-evaluation-the-definitive-guide/), and ultimately improving your LLM through fine-tuning.
+
+This article covers **Phoenix**, which is the open-source solution.
+
+The complexity that is involved in building an LLM application is why observability is so important. Each step of the response generation process needs to be monitored, evaluated and tuned to provide the best possible experience. Not only that, certain tradeoffs might need to be made to optimize for speed, cost, or accuracy. In the context of LLM applications, we need to be able to understand the internal state of the system by examining telemetry data such as LLM Traces.
+
+**Traces** are made up of a sequence of `spans`. A span represents a unit of work or operation (think a span of time). It tracks specific operations that a request makes, painting a picture of what happened during the time in which that operation was executed.
+
+LLM Tracing supports the following span kinds:
+
+![../assets/use_cases/retrieval_augmented_generation_eval_qdrant_arize/arize_spans_types.png](../assets/use_cases/retrieval_augmented_generation_eval_qdrant_arize/arize_spans_types.png)
+
+By capturing the building blocks of your application while it's running, Phoenix can provide a more complete picture of the inner workings of your application.
+
+**Evaluating a Naive RAG using Phoenix**
```

Comment:
I think as the piece is quite long the first part should be left out and maybe some relevant sentences inserted around the other parts where relevant.

This could be the starting point.

Reply:
I am not sure what is the TODO here ?

Reply:
ah alright ! let me revisit !

Reply:
Done , please review!